receiver/apachesparkreceiver/metadata.yaml

```
@@ -4,4 +4,783 @@ status:
+    type: int
+  stage_status:
+    description: The status of the stage for which the metric was recorded.
+    type: string
+    enum:
+      - ACTIVE
+      - COMPLETE
+      - PENDING
+      - FAILED
+  source:
+    description: The source from which data was fetched for the metric.
+    type: string
+    enum:
+      - local
+      - remote
+  location:
+    description: The location of the memory which is quantified in the metric.
+    type: string
+    enum:
+      - on_heap
+      - off_heap
+  executor_id:
+    description: The ID of the executor for which the metric was recorded.
+    type: string
+  job_id:
+    description: The ID of the job for which the metric was recorded.
+    type: int
+  pool_memory_type:
+    description: The type of pool memory for which the metric was recorded.
+    type: string
+    enum:
+      - direct
+      - mapped
+  gc_type:
+    description: The type of the garbage collection performed for the metric.
+    type: string
+    enum:
+      - major
+      - minor
+
+metrics:
+  #stage
+  spark.stage.active_tasks:
+    description: Number of active tasks in this stage.
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: false
+      value_type: int
+    unit: "{ tasks }"
+    attributes:
+      [application_id, application_name, stage_id, attempt_id, stage_status]
+  spark.stage.complete_tasks:
+    description: Number of complete tasks in this stage.
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: true
+      value_type: int
+    unit: "{ tasks }"
+    attributes:
+      [application_id, application_name, stage_id, attempt_id, stage_status]
+  spark.stage.failed_tasks:
+    description: Number of failed tasks in this stage.
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: true
+      value_type: int
+    unit: "{ tasks }"
+    attributes:
+      [application_id, application_name, stage_id, attempt_id, stage_status]
+  spark.stage.killed_tasks:
+    description: Number of killed tasks in this stage.
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: true
+      value_type: int
+    unit: "{ tasks }"
+    attributes:
+      [application_id, application_name, stage_id, attempt_id, stage_status]
+  spark.stage.executor_run_time:
+    description: Amount of time spent by the executor in this stage.
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: true
+      value_type: int
+    unit: ms
+    attributes:
+      [application_id, application_name, stage_id, attempt_id, stage_status]
+  spark.stage.executor_cpu_time:
+    description: CPU time spent by the executor in this stage.
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: true
+      value_type: int
+    unit: ns
+    attributes:
+      [application_id, application_name, stage_id, attempt_id, stage_status]
+  spark.stage.result_size:
```

Seems task related, should it share a namespace?
```suggestion
  spark.stage.task.result_size:
```